charts/jaeger/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.22.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
 type: application
-version: 0.50.0
+version: 0.50.1
```

Comment:
need to bump this to 0.50.2.
https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger/Chart.yaml#L6

Reply:
done